packages/api/src/cli/codegen/languages/typescript/util.ts

```diff
@@ -0,0 +1,74 @@
+import camelCase from 'lodash.camelcase';
+import deburr from 'lodash.deburr';
+import startCase from 'lodash.startcase';
+import { format as prettier } from 'prettier';
+
+export function formatter(content: string) {
+  return prettier(content, {
+    parser: 'typescript',
+    printWidth: 100,
+    singleQuote: true,
+  });
+}
+
+/**
+ * @see {@link https://www.30secondsofcode.org/js/s/word-wrap}
+ */
+export function wordWrap(str: string, max = 88) {
+  return str.replace(new RegExp(`(?![^\\n]{1,${max}}$)([^\\n]{1,${max}})\\s`, 'g'), '$1\n');
+}
+
+/**
+ * Convert a string that might contain spaces or special characters to one that can safely be used
+ * as a TypeScript interface or enum name.
+ *
+ * This function has been adapted and slighty modified from `json-schema-to-typescript`.
+ *
+ * @license MIT
+ * @see {@link https://github.com/bcherny/json-schema-to-typescript}
+ */
+export function toSafeString(str: string) {
```

Comment:
This is a safety net that I pulled over from `json-schema-to-typescript` to ensure that the types we're creating can be used in TS.